packages/edit-site/src/components/sidebar-navigation-screen-library/category-item.js

```diff
@@ -30,12 +36,21 @@ export default function CategoryItem( {
 		return;
 	}
 
+	const ariaLabel = sprintf(
+		/* translators: %1$s is the category name, %2$s is the type (pattern or template part), %3$s is the number of items. */
+		_n( '%1$s (%2$s), %3$s item', '%1$s (%2$s), %3$s items', count ),
```

Comment:
This string would be a bit complicated to translate. It contains a variable part (the type) that may vary in gender on same languages. Ideally, we should try to follow bets practices here and try to provide separate strings with only the count as variable part.